Resources/Prototypes/_ES/Masks/masks.yml

```diff
@@ -129,7 +111,22 @@
   id: ESInfiltrator
   name: es-mask-infiltrator-name
   description: es-mask-infiltrator-desc
-  color: yellow
   mindComponents:
   - type: ESMaskCacheSpawner
     cacheProto: ESCrateCacheTraitorInfiltrator
+
+- type: esMask
+  parent: ESBaseTraitor
+  id: ESRecruit
+  name: es-mask-recruit-name
+  description: es-mask-recruit-desc
```

Comment:
both masks need their own colors (traitor masks were recently changed to have unique colors)